src/main/java/com/rybka/PropertyReader.java

```diff
@@ -0,0 +1,38 @@
+package com.rybka;
+
+import com.rybka.exception.InvalidPropertyException;
+import com.rybka.service.BaseCurrencyExchangeConnector;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+
+public class PropertyReader {
+    private final Map<String, BaseCurrencyExchangeConnector> connectorMap;
+
+    public PropertyReader(Map<String, BaseCurrencyExchangeConnector> connectorMap) {
+        this.connectorMap = connectorMap;
+    }
+
+    public String getPropertyValue() {
```

Comment:
What if we would add another property? 
Make reader more reusable please.

Reply:
The responsibility of PR is: 
read from file and give value by key.

Reply:
Additionally you can try to map properties to the some PropertiesObject. 
From the other side we have Properties class in java, try to utilize it.

Reply:
fixed